### PR TITLE
Import new tokens - 2022-07-12-1617

### PR DESCRIPTION
**Pull request by chuckharmston**

### CODE REVIEW

File: duckduckgo.pot [hunk 1]
```diff
@@ -1878,6 +1878,12 @@ msgctxt "safe search"
 msgid "Don't filter adult content"
 msgstr ""
 
+# smartling.placeholder_format=C
+#. Headline on the Mountain Climber Onboarding slide
+msgctxt "onboarding_slide"
+msgid "Don't let friends get tracked!"
+msgstr ""
```
Comment (chuckharmston) [inline duckduckgo.pot:1885]: @brianhall, it seems like this string (as well as a few more below) that want to ride along on this PR. Any concern with sending these to Smartling?

Comment (brianhall) [reply duckduckgo.pot:1885]: No concerns at all, has been on my to do list. Thank you, @chuckharmston!

**Review by brianhall — APPROVED** [review]

**Comment by GioSensation** [comment]
@chuckharmston I've started the translation job. Please add me to the appropriate Asana task so I can follow up there. Thanks!

**Comment by GioSensation** [comment]
Translations added. Merging now.

**Pull request merged**